frontend/app/controllers/Joiner.scala

```diff
@@ -294,8 +294,10 @@ object Joiner extends Controller with ActivityTracking with PaymentGatewayErrorH
     } yield {
       tier match {
         case t: Tier.Supporter if !upgrade => {salesforceService.metrics.putThankYou(tier)
+          logger.info(s"thank you page displayed ${tier.name}")
           MembersDataAPI.Service.removeBehaviour(request.user)}
-        case t: Tier if !upgrade => salesforceService.metrics.putThankYou(tier)
+        case t: Tier if !upgrade => {salesforceService.metrics.putThankYou(tier)
+          logger.info(s"thank you page displayed ${tier.name}")}
         case _ =>
       }
```

Comment:
Can we move the logging outside the pattern match to avoid repetition? It's also worth logging the identityId so we can identify specific users affected by this issue.